config.yml

```diff
@@ -7427,6 +7428,21 @@ production_recipes:
         material: SPONGE
         amount: 32
         display_name: Bastion
+  Sequencing_Allium:
+    name: Sequencing Allium
+     inputs:
+      Poppy:
+        material: RED_ROSE
+         amount: 8
+      Blue Orchid:
+        material: RED_ROSE
+         amount: 8
+        durablity: 1

```

Comment:
Just make a pull for a fix. No biggie.

On Wed, May 20, 2015, 11:21 AM Akiyama64 <notifications@github.com> wrote:

> In config.yml
> <https://github.com/Civcraft/FactoryMod/pull/65#discussion_r30720229>:
>
> > @@ -7427,6 +7428,21 @@ production_recipes:
> >          material: SPONGE
> >          amount: 32
> >          display_name: Bastion
> > +  Sequencing_Allium:
> > +    name: Sequencing Allium
> > +     inputs:
> > +      Poppy:
> > +        material: RED_ROSE
> > +         amount: 8
> > +      Blue Orchid:
> > +        material: RED_ROSE
> > +         amount: 8
> > +        durablity: 1
>
> I misspelled durability.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Civcraft/FactoryMod/pull/65/files#r30720229>.
>